### ISSUE TITLE
Torchmaster torches might not be working in multiplayer

### ISSUE THREAD

**Issue by osirisgothra** [AllTheMods/VolcanoBlock#23]
I play on a third party server and one of the rewards was  a mega torch. However, it definitely is not working, at least for me.  I even had same-chunk spawns (near opposite edge of chunk while on edge). Someone said it is a nerf, but the tooltip still says 'large area around torch' and they have been that way for years and  years.

Is the range severely nerfed now? Or is this intentional? It seems to not make sense to me since regular torches seem to work the same.

If it isn't a nerf or intentional, it might be worth looking into. I have not been able to discern any situation where it actually does anything more than not having one. It isn't my server, so I can't do any extensive testing. As far as versions, it was set up just a few days ago, so I don't think it is out of date or anything.

**Comment by Shadedstalker2** [comment]
looking at the config it doesnt stop certain mobs from spawning

**Comment by osirisgothra** [comment]
And which config is that? All I see is the agressive mobs scanning is turned off, is this it? If so, do regular vanilla mobs need aggressive checking? But regardless, even one dark block (less than 7) will spawn on an otherwise lit area with a torch in the chunk--doesn't seem like _that_ would need agressive checking. I could understand if the occasional mob spawned when there are many dark blocks but based on my testing the spawn rates are exactly the same.

I tested this by running a small 1-chunk hostile mob farm for a set amount of time. Both with and without the torch. In the passed time, the same amount of materials (within expected differences) were recieved.

If the radius by default is 4 chunks, this shouldn't be a problem. I don't think the items should even be shown or at least marked disabled if it is intended to not work at all. Feral flares do work as expected. I didn't have an opportunity to test the dread lamps yet. And honestly, those I never use anyway.

To sum up, an item marked with a tooltip that states that it "prevents natural spawning of monsters in a big radius around torch" (4 chunks by default) .. if this is intentional, as you can have a mega torch in the same chunk and still get spiders, creepers, etc. - then it should be noted as a disabled item or removed from sight entirely if possible. 

My aim here also is to discern also whether it is something I need to report to the mod maker - but as I cannot reproduce this anywhere but this modpack, I don't have any evidence to do so. This is a multiplayer issue so far, I haven't checked single player as of yet but plan to later today, I'll get back to you on what I find.

**Comment by darklotus781** [comment]
Mega Torches do not seem to prevent mob spawns.  I've had this experience on a single player world.  It's very very easy to reproduce, just place down a mega torch and mobs will spawn where light levels allow it.

**Comment by Draxial** [comment]
We haven't nerfed the mega torch torchmaster afaik, the config shows the range of 64, maybe it's on that server that the config was changed? You mentioned 'someone said it was nerfed', so maybe they mean't on that specific server?

I tested singleplayer with various torches placed and it didn't spawn hostile mobs, just chickens:
![image](https://github.com/AllTheMods/VolcanoBlock/assets/53184989/973f69b0-89f6-4b18-b4c0-ee6e7c6ef1f1)

also tried a dark cobble box with mega torches on top:
![image](https://github.com/AllTheMods/VolcanoBlock/assets/53184989/e367a162-1c4d-431c-9fa7-33e361733510)

(I let the game run for a while, and waited various night and day cycles)

I removed the torches, and immediatelly mobs started spawning:
![image](https://github.com/AllTheMods/VolcanoBlock/assets/53184989/0a5b5ab5-e148-439f-8ff0-e1d123e98528)


just in case, torchmaster doesn't disable spawns from dreadful dirt nor mob spawners themselves, but I'm assuming you made the box with only normal materials right?

**Comment by Draxial** [comment]
Sorry, forgot to get back to you about testing in server as well. I tested in the patreon atmbv server and it's working normally too. Asked a couple members to test too and theirs were working as intended.

Maybe the server you play nerfed them, or disabled their effectiveness?
